app/src/main/res/values-de/help.xml

```diff
@@ -0,0 +1,231 @@
+        <h2>Ringgröße</h2>
+        <p>Es gibt verschiedene Ringgrößen in Monerujo aus denen du wählen kannst. Falls du ein neuer Nutzer bist
+        empfehlen wir bei der Standard Ringgröße zu bleiben (aktuell 7). Die Ringgröße zu erhöhen erhöht die Anzahl der Signaturen in einer Ring-Signatur, 
+		was theoretisch das Erraten der richtigen Signatur erschwert. Eine höhere Ringgröße lässt eine Transaktion jedoch 
+		auf der Blockchain auch herausstechen.</p>
+        <h2>Priorität</h2>
+        <p>Diese Einstellung bestimmt wie zügig deine Transaktion in einem Block auf der Blockchain verarbeitet wird.
+		Eine höhere Priorität bedeutet eine höhere Transaktionsgebühr, eine niedrige Priorität bedeutet eine 
+		niedrigere Transaktionsgebühr. Bitte bedenke, dass eine Transaktion mit einer niedrigen Priorität unter Umständen
+		eine Weile benötigt bis sie auf der Blockchain verarbeitet wird. Die Standard-Priorität
+        ist \"Mittel\".</p>
+        <h1>BTC Senden</h1>
+        <h2>XMR.TO</h2>
+        <p>XMR.TO ist ein Drittanbieter-Service welcher als Tauschbörse von Monero zu Bitcoin fungiert.
+        Wir nutzen die XMR.TO Schnittstelle um Bitcoin Zahlungen in Monerujo zu integrieren. Bitte schaue dir
+        https://xmr.to an und entscheide selbst, ob du diesen Service nutzen willst. Das Monerujo
+        Team gehört nicht zu XMR.TO und kann keinen Support für ihren Service bieten</p>
+        <h2>XMR.TO Tauschkurs<h2>
+        <p>Auf dem \"Menge\" Bildschirm siehst du die aktuellen Werte des XMR.TO Service. Diese beinhalten den aktuellen Tauschkurs
+		sowie die oberen und unteren BTC-Limits. Bitte bedenke, dass dieser Kurs nicht garantiert ist an diesem Punkt.
+		Außerdem siehst du den Wert bis zu dem eine BTC-Zahlung sofort ausgeführt wird 
+		ohne auf XMR-Bestätigungen warten zu müssen (siehe die XMR.TO FAQ für mehr Details). 
+        XMR.TO erhebt keine zusätzlichen Gebühren - wie cool ist das denn?</p>
+        <h2>XMR.TO Auftrag<h2>
+        <p>Auf dem \"Bestätigen\" Bildschirm siehst du den genauen XMR.TO Auftrag. Dieser Auftrag gilt
+        für eine bestimmte Zeit - du siehst einen Countdown auf dem \"Ausgeben\" Button. Der Tauschkurs kann 
+        sich von den vorherigen Angaben unterscheiden.</p>
+        <h2>Geheimer XMR.TO Schlüssel<h2>
+        <p>Da Monerujo nur den Monero-Teil deiner Transaktion verarbeitet kann dein geheimer XMR.TO Schlüssel
+        dazu benutzt werden den Bitcoin-Teil deines Auftrags auf der XMR.TO-Homepage zu verfolgen.</p>
+        <p>Dieser geheime Schlüssel ist für 24h nach Start der Transaktion gültig!</p>
+        <h2>XMR.TO Countdown!</h2>
+        <p>Wenn der Countdown 0 erreicht musst du ein neues Angebot von XMR.TO anfordern indem du zum vorherigen Schritt
+        zurückkehrst und wieder zum \"Bildschirm\" zurückkommst.</p>
+    ]]></string>
+
+    <string name="help_xmrto"><![CDATA[
```

Comment:
this section partially uses different translations than the section above. I feel they are better here.

Reply:
Changed from google Translator to another one. Erm, commit from Rafficer, will change a few things to better fit together